app/components/blacklight/constraints_component.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 module Blacklight
-  class ConstraintsComponent < ::ViewComponent::Base
-    include Blacklight::ContentAreasShim
```

Comment:
Was this exclusion intentional?

Reply:
Added a commit, will squash after review